### PR TITLE
update repository url

### PR DESCRIPTION
**Pull request by simbabque**

### CODE REVIEW

File: dist.ini [hunk 1]
```diff
@@ -9,8 +9,8 @@ copyright_year   = 2010
 [MetaJSON]
 
 [MetaResources]
-repository.url    = http://github.com/melo/app-httpthis.git
-repository.web    = http://github.com/melo/app-httpthis/
+repository.url    = http://github.com/melo/app-htttpthis.git
+repository.web    = http://github.com/melo/app-htttpthis/
```
Comment (melo) [inline dist.ini:13]: Do you mind updating to HTTPS also?

Comment (melo) [reply dist.ini:13]: Also, I just noticed, that the problem is that my Github URL has a big typo, extra 't'... Maybe I can just rename the repository :)... That is the correct fix.

Comment (simbabque) [reply dist.ini:13]: Yeah I thought you might have renamed this later, for some reason. That's why I didn't point it out. Just rename it then. That'll fix it.

**Comment by melo** [comment]
#2 solved this

**Pull request closed without merging**